framework/model/module/module.go

```diff
@@ -49,6 +49,11 @@ type configH struct {
 // ModuleOptions carries the framework context for setting a module.
 // For fields marked with REQUIRED, the framework will ensure their existence,
 // and the module can be used directly.
+// NOTE: the Manager and the LeaderElectionCbs share the election status.
```

Comment:
put here or the `Module` interface?

Reply:
`Moudle` interface is a more reasonable location, but there is a lack of context about elections.